src/layman/map/util.py

```diff
@@ -123,8 +124,10 @@ def patch_map(workspace, mapname, task_options, start_at):
 
 
 def delete_map(workspace, mapname, kwargs=None):
+    publ_uuid = get_publication_uuid(workspace, MAP_TYPE, mapname)
     sources = get_sources()
     call_modules_fn(sources[::-1], 'delete_map', [workspace, mapname], kwargs=kwargs)
+    delete_publication_uuid(workspace, MAP_TYPE, mapname, publ_uuid)
```

Comment:
I suggest this method to either move to redis package or rename.

Reply:
Renamed